UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -515,6 +515,64 @@ public static List<Protein> LoadProteinFasta(string proteinDbLocation, bool onTh
             return result;
         }
 
+        /// <summary>
+        /// Merges proteins of the same accession and sequence. Considers contaminants separately.
```

Comment:
Merge proteins that have the same accession, sequence and contaminant designation